app/assets/stylesheets/custom.css.scss

```diff
@@ -75,66 +75,73 @@ h1 {
 #flash_notice {
   font-family: monospace;
   text-style: bold;
-  text-color: red;
   opacity: .8;
   text-align: center;
   margin-right: 60%;
   padding:4px;
   border-radius: 100px;
-  color: cornsilk;
+  color: #fff8dc;
   background-color: darkgrey;
 }
 
 #flash_errors{
   font-family: monospace;
   text-style: bold;
-  text-color: red;
   opacity: .8;
   text-align: center;
   margin-right: 60%;
   padding:4px;
   border-radius: 100px;
-  color: cornsilk;
+  color: #fff8dc;
```

Comment:
Color literals like `#fff8dc` should only be used in variable declarations; they should be referred to via variable everywhere else.